gammapy/data/observation_stats.py

```diff
@@ -0,0 +1,189 @@
+        return sigma
+
+    @staticmethod
+    def _get_on_events(obs, target):
+        """Number of ON events in the region of interest (`int`)
+        """
+        print(target)
+        idx = target.on_region.contains(obs.events.radec)
+        on_events = obs.events[idx]
+        return len(on_events)
+
+    @classmethod
+    def stack(cls, stats_list):
+        """Stack statistics from a list of 
+        `~gammapy.data.ObservationStats` and returns a new instance 
+        of `~gammapy.data.ObservationStats`
+
+        Parameters
+        ----------
+        stats_list : list
+            List of observation statistics `~gammapy.data.ObservationStats`
+      
+        Returns
+        -------
+        total_stats : `~gammapy.data.ObservationStats`
+            Statistics for stacked observation 
+        """
+        n_on = 0
+        n_off = 0
+        a_on = 0
+        a_on_backup = 0
+        a_off = 0
+        a_off_backup = 0
+        alpha = 0
+        alpha_backup = 0
+        livetime = 0
+        obs_id = list()
+        for stats in stats_list:
+            livetime += stats.livetime
+            n_on += stats.n_on
+            n_off += stats.n_off
+            a_on += stats.a_on * stats.n_off
+            a_on_backup += stats.a_on * stats.livetime.value
+            a_off += stats.a_off * stats.n_off
+            a_off_backup += stats.a_off * stats.livetime.value
+            alpha += stats.alpha * stats.n_off
+            alpha_backup += stats.alpha * stats.livetime.value
+            obs_id.append(stats.obs_id)
+
+        a_on /= n_off
+        a_off /= n_off
+        alpha /= n_off
+
+        # if no off events the weighting of alpha is done
+        # with the livetime
+        if n_off == 0:
+            alpha = alpha_backup / livetime.value
+            a_on_backup = a_on_backup / livetime.value
+            a_off_backup = a_off_backup / livetime.value
+
+        total_stats = cls(
+            n_on=n_on,
+            n_off=n_off,
+            a_on=a_on,
+            a_off=a_off,
+            obs_id=obs_id,
+            livetime=livetime,
+            alpha=alpha
+        )
+        return total_stats
+
+    def __add__(self, other):
+        """Add statistics from two observations
+        and returns new instance of `~gammapy.data.ObservationStats`
+        """
+        return ObservationStats.stack(self, other)
+
+    def __str__(self):
+        """Observation statistics report (`str`)
+        """
+        ss = '*** Observation summary report ***\n'
+        ss += 'Observation Id: {}\n'.format(self.obs_id)
+        ss += 'Livetime: {}\n'.format(self.livetime.to(u.h))
+        ss += 'On events: {}\n'.format(self.n_on)
+        ss += 'Off events: {}\n'.format(self.n_off)
+        ss += 'Alpha: {}\n'.format(self.alpha)
+        ss += 'Bkg events in On region: {}\n'.format(self.background)
+        ss += 'Excess: {}\n'.format(self.excess)
+        ss += 'Gamma rate: {}\n'.format(self.n_on / self.livetime.to(u.min))
```

Comment:
Maybe make the gamma and background rate properties on the class?
I think they are useful to have, and you're printing them, so it seems to be common info someone might want to get.